app/views/places/_map_attribution.html.haml

```diff
@@ -2,5 +2,3 @@ Map data &copy;
 = link_to "OpenStreetMap", "https://openstreetmap.org"
 contributors under
 = link_to "ODbL", "https://www.openstreetmap.org/copyright"
-| Imagery &copy;
-= link_to "CloudMade", "https://cloudmade.com"
```

Comment:
this company used to produce the leaflet widget, but that staff member moved elsewhere in like 2012 and took the code with them and made it open source. the credit is no longer needed